setup/pi/configure.sh

```diff
@@ -363,8 +363,11 @@ install_push_message_scripts /root/bin
 
 check_archive_configs
 
-echo "ARCHIVE_HOST_NAME=$archiveserver" > /root/teslausb.conf
-echo "ARCHIVE_DELAY=${archivedelay:-20}" >> /root/teslausb.conf
+configFile = /root/teslausb.conf
+
+echo "ARCHIVE_HOST_NAME=$archiveserver" > configFile
```

Comment:
That needs to be `$configFile`, otherwise it's just going to write it to a file named configFile in the current directory.

Reply:
good catch